docs/src/man/query_package.md

```diff
@@ -0,0 +1,69 @@
+# Query package
+
+The [Query.jl](https://github.com/davidanthoff/Query.jl) package provides advanced data manipulation capabilities for `DataFrames` (and many other data structures). This section provides a short introduction to the package, the [Query.jl](https://github.com/davidanthoff/Query.jl) [documentation](http://www.david-anthoff.com/Query.jl/stable/) has a more comprehensive documentation of the package.
```

Comment:
I would make this a single link to the docs, as there's already a link to the package project above (same remark for the last paragraph of the page).